### DISCUSSION TITLE
Retirement of custom default result sources in Microsoft Search for modern search experiences

### DISCUSSION THREAD

**Discussion by dowsetts101** [microsoft-search/pnp-modern-search#2849] [Q&A] [answered]
Hi, I've only just started using the pnp modern search webparts and only understand a very limited amount about them, but have been starting to get them working nicely, so very excited to see what I can do with them.

Our company admin recently shared a message centre post about the "**Retirement of custom default result sources in Microsoft Search for modern search experiences**". "_Starting mid-April 2023, customers using a custom default result source for KQL rewrite of the modern search experience should move KQL rewrites from those sources over to the vertical administration configuration_"

I'm trying to work through the potential impact of this change to my search webparts. I'm thinking that as I'm using SharePoint search in my webparts rather than Microsoft Search that it wont affect things...but I'm still a babe in the woods with regards to search, so wanted to see if anyone had any better informed thoughts on all this stuff. Thanks in advance.

**Comment by kasperbolarsen** [comment, marked as answer]
Please see https://www.techmikael.com/2023/03/retirement-of-custom-default-result.html?m=1

**Reply by dowsetts101** [reply]
Fantastic thanks for that. Appreciate the prompt response. Sounds like nothing too much to worry about from a PnP Search webpart perspective!!!

**Reply by kasperbolarsen** [reply]
Mikael says no , in a thread on Twitter

**Reply by wobba** [reply]
Nothing to worry about, the MC post is only about modern search experiences delivered by Microsoft. Should not impact many at all, and no API scenarios, but posts have to be sent out :) Will likely be some more text added to the note section.

If you read the "What you can do to prepare" section, it should be clear what it impacts.

**Comment by jagdaleprashant** [comment]
Hi, we are using PNP search web part and data source is SharePoint , KQL query written on top of that , so do we have any impact due to this retirement?

**Reply by wobba** [reply]
No. If you read the post it's for modern OOB search experiences only. These web parts are not part of the product offering, thus not affected.
